### DISCUSSION TITLE
TMT P

### DISCUSSION THREAD

**Discussion by roushrsh** [MannLabs/alphadia#221] [Questions / Support] [answered]
Hi, 

Looking try out your tool. I discussed with one of the presenters at ASMS and it seems I should be able to search my TMT pro labeled samples with this. Wondering which mod I should use? (Mass added is the 304 dalton of course).

Thanks!

**Comment by GeorgWa** [comment, marked as answer]
Hi, I think we talked briefly. You are looking for the Mannlabs/alphadia repository and will have to provide it as fixed modification.

There is no quantification of reporters or complement fragments.

**Reply by roushrsh** [reply]
Awesome, thanks so much, will try it out.

**Reply by roushrsh** [reply]
Have it working for unlabeled.
But I don't see any option for tmt pro. What should I write in for the fixed mod section?
Thanks!

**Reply by GeorgWa** [reply]
Have a look at: https://github.com/MannLabs/alphabase/blob/main/alphabase/constants/const_files/modification.tsv

**Reply by jalew188** [reply]
If you know the unimod ID, you can search unimod ID in the tsv table.

**Reply by roushrsh** [reply]
Thank you!
